<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VitaWidget</class>
 <widget class="QWidget" name="VitaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VitaWidget</string>
  </property>
  <widget class="QRadioButton" name="radioClassic">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>10</y>
     <width>78</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vita Classic</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioMaster">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>40</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Vita 3D Master</string>
   </property>
  </widget>
  <widget class="QComboBox" name="index">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A3,5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="chroma_hue">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>40</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="idx_label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>10</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Цвят:</string>
   </property>
  </widget>
  <widget class="QLabel" name="c_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>40</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Нюанс:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
